src/Xamarin.Android.Build.Tasks/Utilities/MarshalMethodsAssemblyRewriter.cs

```diff
@@ -34,13 +35,40 @@ public void Rewrite (DirectoryAssemblyResolver resolver)
 				unmanagedCallersOnlyAttributes.Add (asm, CreateImportedUnmanagedCallersOnlyAttribute (asm, unmanagedCallersOnlyAttributeCtor));
 			}
 
+			Console.WriteLine ();
```

Comment:
Shouldn't this and other `Console.WriteLine()`s here use `Log.LogDebugMessage()` instead?

Reply:
Not all of them will stay. The next PR in the series cleans that up.